docs/registration.adoc

```diff
@@ -0,0 +1,125 @@
+:toc: left
+:toclevels: 3
+:sectanchors: true
+:sectids: true
+:source-highlighter: rouge
+:icons: font
+
+= Registration
```

Comment:
The staking provider <> operator mapping registration is something the staking provider should do. I assume the entire `registration.adoc` is for staking providers and focus on the registration of this mapping. If so, then I think *Stake* and *Authorize Applications* should be expressed as *Prerequirements*.

Reply:
Actually, the `registration.adoc` is not only for the Staking Providers. The idea is that it will describe the whole process. It describes only the Staking Providers' duties as we need to document them now, leaving the staking part blank for now.

We should have a walk-through for all the actors.

Reply:
In the Staking Provider bundle we link to the `Register Operator` section 
https://github.com/keep-network/keep-core/pull/3216/files#diff-3b6c148512a584866c6b1e32dd5b424e19b1d63638be5a8e1d38d1eb48712f12R60

Reply:
I think we should separate staking and authorization out of this section. Registration is specifically for the staking providers and is about mapping two addresses. Staking and authorization are not a part of the registration.